site/src/App.css

```diff
@@ -97,6 +97,7 @@ a:visited {
   border: 1px solid white;
   background: #202020;
   padding: 10px;
+  min-width: 250px;
```

Comment:
This feels like an ugly hack, but my CSS is rusty, and without it the text was displaying as:

```
Line
15
Column
58
```

250px seemed reasonable based on making most of the enum values for flags showing up on one line

Reply:
My css sucks too. LGTM 😂